Makefile

```diff
@@ -43,6 +44,9 @@ $(BIN_DIR)/minishift:
 	@cd $(BIN_DIR) && \
 	curl -LO --progress-bar $(MINISHIFT_LATEST_URL) && \
 	tar xzf $(ARCHIVE_FILE)
+	@echo "Moving minishift binary to $(BIN_DIR)"   
```

Comment:
Update line `42` with `@echo "Downloading latest minishift binary at $(BIN_DIR)/minishift"`. This one is not required.

Reply:
@anjannath , CI is also failing. The another cd to $(BIN_DIR) is not working fine. Check